metadata-ingestion/archived/source_docs/s3_data_lake.md

```diff
@@ -74,11 +74,13 @@ sink:
 Note that a `.` is used to denote nested fields in the YAML recipe.
 
 | Field                                                | Required                 | Default                                   | Description                                                                                                                                                                                                    |
-| ---------------------------------------------------- | ------------------------ | ----------------------------------------- | -------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
-| `path_spec.include`                                  | ✅                       |                                           | Path to table (s3 or local file system). Name variable {table} is used to mark the folder with dataset. In absence of {table}, file level dataset will be created. Check below examples for more details.      |
-| `path_spec.exclude`                                  |                          |                                           | list of paths in glob pattern which will be excluded while scanning for the datasets                                                                                                                           |
-| `path_spec.table_name`                               |                          | {table}                                   | Display name of the dataset.Combination of named variableds from include path and strings                                                                                                                      |
-| `path_spec.file_types`                               |                          | ["csv", "tsv", "json", "parquet", "avro"] | Files with extenstions specified here (subset of default value) only will be scanned to create dataset. Other files will be omitted.                                                                           |
+|------------------------------------------------------|--------------------------|-------------------------------------------|----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| `path_specs`                                         | ✅                        |                                           | List of PathSpec -> see below                                                                                                                                                                                  |
```

Comment:
Can we use the new decorator way to populate the config specific documentation, instead of doing it manually here?